platform/onebox/backup_and_restore/ceph/export.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+: '
+This scipt will export all data stored in blob storage from target environments.
+# $1 - json file with credentials
+# $2 - path to export
+Script will generate folders like path/ceph/bucket1 path/ceph/bucket2 each containing data from 1 bucket
+[Script Version -> 21.4]
```

Comment:
No, but when we make any change to the script we will update it. It will be easier to check any issues that users face. This is just script version which we are keeping same as release versions